OpenIPSL/Electrical/ThreePhase/Buses/MeasurementBus.mo

```diff
@@ -19,34 +19,34 @@ model MeasurementBus
   SI.ReactivePower Qb(displayUnit="Mvar") "Reactive Power suplyed by the Infinite bus (phase b) (MVAr)";
   SI.ActivePower Pc(displayUnit="MW") "Active Power suplyed by the Infinite bus (phase c) (MW)";
   SI.ReactivePower Qc(displayUnit="Mvar") "Reactive Power suplyed by the Infinite bus (phase c) (MVAr)";
-  Modelica.Blocks.Interfaces.RealOutput pa = Pa/S_b annotation (Placement(transformation(
+  Modelica.Blocks.Interfaces.RealOutput pa = Pa/S_b "[pu, system base]" annotation (Placement(transformation(
         extent={{-24,-24},{24,24}},
         rotation=-90,
         origin={52,-124})));
-  Modelica.Blocks.Interfaces.RealOutput pb = Pb/S_b  annotation (Placement(transformation(
+  Modelica.Blocks.Interfaces.RealOutput pb = Pb/S_b "[pu, system base]" annotation (Placement(transformation(
         extent={{-24,-24},{24,24}},
         rotation=-90,
         origin={2,-124})));
-  Modelica.Blocks.Interfaces.RealOutput pc = Pc/S_b  annotation (Placement(transformation(
+  Modelica.Blocks.Interfaces.RealOutput pc = Pc/S_b "[pu, system base]" annotation (Placement(transformation(
         extent={{-24,-24},{24,24}},
         rotation=-90,
         origin={-50,-124})));
 
-  Modelica.Blocks.Interfaces.RealOutput qa = Qa/S_b annotation (Placement(transformation(
+  Modelica.Blocks.Interfaces.RealOutput qa = Qa/S_b "[pu, system base]" annotation (Placement(transformation(
         extent={{-24,-24},{24,24}},
         rotation=-90,
         origin={52,-124}), iconTransformation(
         extent={{24,-24},{-24,24}},
         rotation=-90,
         origin={60,110})));
-  Modelica.Blocks.Interfaces.RealOutput qb = Qb/S_b  annotation (Placement(transformation(
+  Modelica.Blocks.Interfaces.RealOutput qb = Qb/S_b "[pu, system base]" annotation (Placement(transformation(
         extent={{-24,-24},{24,24}},
         rotation=-90,
         origin={2,-124}), iconTransformation(
         extent={{24,-24},{-24,24}},
         rotation=-90,
         origin={0,110})));
-  Modelica.Blocks.Interfaces.RealOutput qc = Qc/S_b  annotation (Placement(transformation(
+  Modelica.Blocks.Interfaces.RealOutput qc = Qc/S_b "[pu, system base]" annotation (Placement(transformation(
```

Comment:
```suggestion
  Modelica.Blocks.Interfaces.RealOutput qc(unit="1") = Qc/S_b "[pu, system base]" annotation (Placement(transformation(
```

Reply:
Shame that you did not simply accept the already provided suggestions by simply clicking when you still could ;-)

Reply:
:(